app/assets/stylesheets/components/_topic-list.scss

```diff
@@ -4,10 +4,20 @@
   margin-bottom: $gutter-one-third;
 }
 
-.app-c-topic-list__link {
+.app-c-topic-list__link--no-underline {
   text-decoration: none;
```

Comment:
Out of scope but - can we remove this? 
I feel like [these](https://www.gov.uk/topic/environmental-management) should have an underline as they're links